test/basic.test.js

```diff
@@ -48,16 +48,12 @@ test('no error when favicons missing from page', async () => {
   }
 })
 
-test('favicons', async () => {
-  const url = 'https://www.bbc.com/news/uk-england-somerset-68179350'
-  try {
-    const metadata = await urlMetadata(url)
-    expect(metadata.favicons.length).toBe(5)
-    expect(metadata.favicons[0].rel).toBe('apple-touch-icon')
-    // Safari pinned tab 'mask-icons' can have 'color' attribute:
-    expect(metadata.favicons[4].rel).toBe('mask-icon')
-    expect(metadata.favicons[4].color).toBe('#000000')
-  } catch (err) {
-    expect(err).toBe(undefined)
-  }
-})
+// test('favicons', async () => {
+//   const url = 'https://www.bbc.com/news/uk-england-somerset-68179350'
+//   const metadata = await urlMetadata(url)
+// expect(metadata.favicons.length).toBe(5)
+// expect(metadata.favicons[0].rel).toBe('apple-touch-icon')
+// Safari pinned tab 'mask-icons' can have 'color' attribute:
+// expect(metadata.favicons[4].rel).toBe('mask-icon')
+// expect(metadata.favicons[4].color).toBe('#000000')
+// })
```

Comment:
This one started failing for me, I guess the website changed?

I guess comitting the HTML responses into some mocks folder would be a way to go here. Test via the HTML response feature or mock.

chatGPT:

Create a Manual Mock for Fetch:

If you're using node-fetch, for example, you'd place your mock in a __mocks__ folder at the root of your project (or wherever your jest.config.js is located) to mirror the module structure. For node-fetch, the path would be __mocks__/node-fetch.js.

Implement the Mock:

Inside __mocks__/node-fetch.js, you can define what you want your mocked fetch to return when called during tests:

```js
// __mocks__/node-fetch.js
module.exports = jest.fn(() =>
  Promise.resolve({
    json: () => Promise.resolve(/* Your mocked JSON-LD data here */),
  })
);
```
Use Mock in Your Test:

In your test file, you need to tell Jest to use the mock instead of the actual implementation:

```js
jest.mock('node-fetch'); // Replace 'node-fetch' with whatever library `urlMetadata` uses to make HTTP requests.
```
```js
test('retrieves multiple json-ld objects if url has it', async () => {
  // Your test implementation remains the same
});
```